paasta_tools/contrib/rightsizer_soaconfigs_update.py

```diff
@@ -121,7 +133,9 @@ def main(args):
     extra_message = get_extra_message(report["search"])
     config_source = args.source_id or args.csv_report
 
-    results = get_recommendations_by_service_file(report["results"])
+    results = get_recommendations_by_service_file(
```

Comment:
I think it would be a more logical code structure if you did the filtering separately, before the get_recommendations_by_service_file transform.